src/unitycatalog/ai/utils.py

```diff
@@ -113,3 +141,159 @@ def validate_full_function_name(function_name: str) -> FullFunctionName:
             f"Invalid function name: {function_name}, expecting format <catalog_name>.<schema_name>.<function_name>."
         )
     return FullFunctionName(catalog_name=splits[0], schema_name=splits[1], function_name=splits[2])
+
+
+def uc_type_json_to_pydantic_type(uc_type_json: Union[str, Dict]) -> Type:
+    """
+    Convert Unity Catalog type json to Pydantic type.
+
+    For simple types, the type json is a string representing the type name. For example:
+        "STRING" -> str
+        "INTEGER" -> int
+    For complex types, the type json is a dictionary representing the type. For example:
+        {"type": "array", "elementType": "STRING", "containsNull": true} -> List[Optional[str]]
+
+    Args:
+        uc_type_json (Union[str, Dict]): The Unity Catalog function input parameter type json.
```

Comment:
```suggestion
        uc_type_json: The Unity Catalog function input parameter type json.
```

because the type hint is present